cmd/manager/main.go

```diff
@@ -36,6 +37,8 @@ var (
 	metricsHost               = "0.0.0.0"
 	metricsPort         int32 = 8383
 	operatorMetricsPort int32 = 8686
+
+	osStat = os.Stat
```

Comment:
This was done so that I can mock it in TestCheckIfRequiredFilesExist in main_test.go. Check [this](TestCheckIfRequiredFilesExist). Let me know if you can suggest any other way to mock os.Stat

Reply:
The link is broken. 

Even if you want it to mock it this way, you can just use `os.Stat`, save it, overwrite, run your tests and restore it. I don't see why you need the global variable.